.npmignore

```diff
@@ -1,5 +1,7 @@
-examples/
-src/
+examples/*
+!examples/js/
+src/*
+!src/extras/
```

Comment:
You are right, bower package currently doesn't include the extras. However, I think extras has many useful components that are very useful. It's a shame that npm doesn't allow multiple packages from the same git repo, otherwise best solution would be to have three separate packages. But since that is not possible, I think it is best to have them included in the main repo package rather than separate forks that may or may not be maintained down the line.